daft/dataframe/dataframe.py

```diff
@@ -2532,6 +2532,18 @@ def to_pydict(self) -> Dict[str, List[Any]]:
         assert result is not None
         return result.to_pydict()
 
+    @DataframePublicAPI
+    def to_pylist(self) -> List[Any]:
+        """Converts the current Dataframe into a python list.
```

Comment:
Can we add some example code here? You may reference other docstrings for examples. Also we should add this into our documentation (under `dataframe.rst`)

Reply:
Added example to doc string and added to_pylist to `dataframe.rst`